v3/src/hooks/use-drop-handler.ts

```diff
@@ -7,9 +7,10 @@ export interface IDropHandler {
   onImportDataSet?: (data: IDataSet) => void
   onImportDocument?: (file: File) => void
   onHandleUrlDrop?: (url: string) => void
+  setIsDragOver?: (isDragOver: boolean) => void
```

Comment:
```suggestion
  onSetIsDragOver?: (isDragOver: boolean) => void
```
Conventional naming for callbacks.